.github/workflows/tests.yaml

```diff
@@ -80,10 +79,10 @@ jobs:
           if ( cat conda.txt | grep -q pygeos  )
           then
             echo "Setting HAS_PYGEOS=1"
-            echo '::set-env name=HAS_PYGEOS::1'
+            echo "HAS_PYGEOS=1" >> $GITHUB_ENV
```

Comment:
Yes, this is indeed the new syntax for this (ran into the same issue when converting pygeos CI to github actions: https://github.com/pygeos/pygeos/blob/a524ae4eb590e736307c3054b30b9cd835eafb91/.github/workflows/test-linux.yml#L79)

Reply:
This deprecation will likely cause a massive wave of failures today. I had to fix conda setup in two repos already and I expect more will show up once something triggers GHA.